docs/src/test/scala/docs/http/scaladsl/server/directives/CustomDirectivesExamplesSpec.scala

```diff
@@ -77,4 +79,31 @@ class CustomDirectivesExamplesSpec extends RoutingSpec {
     //#flatMap-0
   }
 
+  "scratch" in {
+    //#scratch
+    object HostnameAndPort extends Directive[(String, Int)] {
+
+      private val hostnameAndPort =
+        textract(ctx => {
+          val auth = ctx.request.uri.authority
```

Comment:
Nitpicking here, but `auth` reminds me too much of Authentication or Authorization, maybe rename it to `authority`?

Reply:
Good advice, `authority ` would be better!